libgalois/include/katana/ExtraTraits.h

```diff
@@ -58,6 +58,13 @@ struct is_serializable {
       has_serialize<T>::value || is_memory_copyable<T>::value;
 };
 
+//! Indicates if T is std::map or std::unordered_map
+template <typename, typename = void>
+struct is_map_type : std::false_type {};
```

Comment:
All the other type traits, omit `_type` in their name; Can you call this `is_map` or `is_associative_container`?

Technically, you might as well make a small test here too (map and unordered map are `is_map`. This will at least provide some coverage on this new code.

Reply:
Thank you for the comment. I added a simple test. Could you please check if this is what you want?

Reply:
Yes, those tests look good.